src/OmniSharp.MSBuild/ProjectFile/ProjectFileInfo.ProjectData.cs

```diff
@@ -187,6 +192,7 @@ private ProjectData()
                 AnalyzerConfigFiles = analyzerConfigFiles.EmptyIfDefault();
                 ReferenceAliases = referenceAliases;
                 ProjectReferenceAliases = projectReferenceAliases;
+                FileInclusionGlobs = fileInclusionGlobs;
             }
 
             public static ProjectData Create(MSB.Evaluation.Project project)
```

Comment:
Would there be any benefit in including the globs as part of the data returned by this method as well?

Reply:
Sorry for the late reply! I've investigated a bit, and it seems that this codepath is used when solution contains projects outside the target workspace folder. Since they do get added to the workspace, it's probably worth a fix